smart_card_connector_app/src/window.css

```diff
@@ -127,3 +127,28 @@ body {
   margin-right: 15px;
   text-decoration: none;
 }
+
+#readers-list > li {
+  display: flex;
+  align-items: center;
+}
+
+.circle {
```

Comment:
CSS class names should explain the meaning, but not describe the actual visual styling.
(This comment relates to "circle", "yellow", "red", "green".)